backend/src/main/java/com/scc/app/service/AppointmentService.java

```diff
@@ -70,4 +70,36 @@ public Appointment getAppointmentById(Long id) {
 
         return appointmentRepository.findByUserId(washingStationId);
     }
+
+    public void update(Appointment appointmentUpdated) {
```

Comment:
Shouldn't the `update` methods return the updated entries?

Reply:
fixed